frontend/packages/knative-plugin/src/components/sink-pubsub/__tests__/SinkPubsub.spec.tsx

```diff
@@ -30,14 +40,14 @@ describe('SinkPubsub', () => {
   it('should render Formik child with label move Subscription for Subscription', () => {
     const formikFormRender = pubsubForm.find(Formik).get(0).props;
     expect(formikFormRender.children).toHaveLength(1);
-    expect(formikFormRender.children().props.labelTitle).toBe('Move Subscription');
+    expect(formikFormRender.children().props.labelTitle).toBe(`${i18nNS}~Move {{sourceKind}}`);
```

Comment:
value for `sourceKind` ?

Reply:
@debsmita1 this for spec , we don't need

Reply:
okay, got it